forge-gui-desktop/pom.xml

```diff
@@ -118,7 +118,7 @@
                         </goals>
                         <configuration>
                             <!-- TODO: insert placeholder for latest version tag -->
-                            <fromRef>forge-1.6.56</fromRef>
+                            <fromRef>forge-1.6.55</fromRef>
```

Comment:
was this change on purpose?

Reply:
yes, that's the latest tag we have available so it needs to start from there